frontend/src/components/profile/info/BadgeCard.tsx

```diff
@@ -128,14 +151,70 @@ export const BadgeCard = ({ user }: { user: User }) => {
         badges.push(<CustomBadge badge={badge} handleBadgeClick={handleBadgeClick} />);
     }
 
+    for (const polgarBadges of getIneligiblePolgarBadgeList(user)) {
+        ineligiblePolgarBadges.push(
+            <CustomBadge
+                badge={polgarBadges}
+                handleBadgeClick={handleBadgeClick}
+                isBlocked={true}
+            />,
+        );
+    }
+
+    for (const cohortBadge of getIneligibleCohortBadgeList(user)) {
+        inelgibleCohortBadges.push(
+            <CustomBadge
+                badge={cohortBadge}
+                handleBadgeClick={handleBadgeClick}
+                isBlocked={true}
+            />,
+        );
+    }
+
+    for (const anongameBadges of getIneligibleBadgeList(user, BadgeType.AnnotateGames)) {
+        ineligibleAnnoBadges.push(
+            <CustomBadge
+                badge={anongameBadges}
+                handleBadgeClick={handleBadgeClick}
+                isBlocked={true}
+            />,
+        );
+    }
+
+    for (const gameBadges of getIneligibleBadgeList(user, BadgeType.ClassicalGames)) {
+        ineligibleGameBadges.push(
+            <CustomBadge
+                badge={gameBadges}
+                handleBadgeClick={handleBadgeClick}
+                isBlocked={true}
+            />,
+        );
+    }
+
     if (badges.length === 0) {
         return null;
     }
 
     return (
         <>
             <Card>
-                <CardHeader title='Badges' />
+                <Stack
+                    direction='row'
+                    justifyContent='space-between'
+                    alignItems='center'
+                    px={2}
+                    pt={2}
+                >
+                    <CardHeader title='Badges' sx={{ p: 0 }} />
```

Comment:
I think we may want more padding between the title and the badges. It looks a bit cramped, especially if the user has graduated, and the glow is almost on top of the title:

<img width="440" alt="Screenshot 2025-02-15 at 10 30 05 AM" src="https://github.com/user-attachments/assets/37878545-a325-4ba6-8dd8-02eab5cdbca2" />